cradle-cassandra/src/test/java/com/exactpro/cradle/cassandra/resultset/PagesInIntervalIteratorProviderTest.java

```diff
@@ -33,30 +33,30 @@ public class PagesInIntervalIteratorProviderTest {
 
     @Test
     public void defaultPageVsIncludedInterval() {
-        PageInfo pageInfo = new PageInfo(new PageId(bookId, Instant.EPOCH, "test-page"), Instant.EPOCH, null, null);
+        PageInfo pageInfo = new PageInfo(new PageId(bookId, Instant.EPOCH, "test-page"), null, null);
 
-        assertTrue(checkInterval(pageInfo, pageInfo.getStarted(), Instant.now()), "in.st = p.st, in.en = p.en");
-        assertTrue(checkInterval(pageInfo, pageInfo.getStarted().plusNanos(1), Instant.now()), "in.st = p.st + 1, in.en = p.en");
-        assertTrue(checkInterval(pageInfo, pageInfo.getStarted(), pageInfo.getStarted()), "in.st = p.st, in.en = p.st");
-        assertTrue(checkInterval(pageInfo, pageInfo.getStarted().minusNanos(1), pageInfo.getStarted()), "in.st = p.st - 1, in.en = p.st");
+        assertTrue(checkInterval(pageInfo, pageInfo.getId().getStart(), Instant.now()), "in.st = p.st, in.en = p.en");
```

Comment:
I would keep `getStarted` method in PageInfo and simply delegate call to `getId(),getStart()` for backward compatibility